docs/connecting.asciidoc

```diff
@@ -6,451 +6,104 @@ This page contains the information you need to connect and use the Client with
 
 **On this page**
 
-* <<authentication, Authentication options>>
-* <<client-usage, Using the client>>
-
-
-[discrete]
-[[authentication]]
-=== Authentication
-
-This section contains code snippets to show you how to connect to various {es} 
-providers.
+* <<auth-ec, Elastic Cloud>>
+* <<auth-http, Security by default (HTTPS)>>
 
 [discrete]
 [[auth-ec]]
 ==== Elastic Cloud
```

Comment:
If you delete line 15 with the 3rd level section, we need to adjust the rest of the section IDs as well.

```suggestion
=== Elastic Cloud
```